core/eolearn/tests/test_core_tasks.py

```diff
@@ -164,6 +168,97 @@ def test_initialize_feature(self):
         self.assertTrue(new_names < set(patch.data), msg=fail_msg)
         self.assertTrue(all(patch.data[key].shape == patch.data['bands'].shape for key in new_names))
 
+    def test_move_feature(self):
+        patch_src = EOPatch()
+        patch_dst = EOPatch()
+
+        shape = (10, 5, 5, 3)
+        size = np.product(shape)
+
+        shape_timeless = (5, 5, 3)
+        size_timeless = np.product(shape_timeless)
+
+        data = [np.random.randint(0, 100, size).reshape(*shape) for i in range(3)] + \
+               [np.random.randint(0, 100, size_timeless).reshape(*shape_timeless) for i in range(2)]
+
+        features = [(FeatureType.DATA, 'D1'),
+                    (FeatureType.DATA, 'D2'),
+                    (FeatureType.MASK, 'M1'),
+                    (FeatureType.MASK_TIMELESS, 'MTless1'),
+                    (FeatureType.MASK_TIMELESS, 'MTless2')]
+
+        for feat, dat in zip(features, data):
+            patch_src = AddFeature(feat)(patch_src, dat)
+
+        patch_dst = MoveFeature(features)(patch_src, patch_dst)
+
+        for i, (ftype, fname) in enumerate(features):
```

Comment:
Also in places like this you can replace `(ftype, fname)` with `feature` and use `patch_dst[feature]`

Reply:
Done.